tests/conftest.py

```diff
@@ -159,3 +159,30 @@ def s3_bucket(request: pytest.FixtureRequest):
         return 'my-bucket'
     else:
         return _get_option(request.config, 's3_bucket')
+
+
+# WARNING: These fixtures should be copied if used in a test, as they are session scope to avoid
+# multiple calls to the HuggingFace hub
+@pytest.fixture(scope='session')
```

Comment:
Can you explain this a bit more? I don't quite get what's going on

Reply:
When we use `scope="session"`, the fixture is only created once and the same object is reused for all tests. By doing this we only make one call to the hugging face hub, and then the tests that need this fixture just make a copy, which is cheaper than the call to the hub. And the tests have to make a copy because they modify the object.

Reply:
haha def not dumb, I will make the comment more clear before this PR goes in